src/script/common/c_content.cpp

```diff
@@ -824,6 +827,10 @@ void read_content_features(lua_State *L, ContentFeatures &f, int index)
 	getstringfield(L, index, "node_dig_prediction",
 		f.node_dig_prediction);
 
+	// How much the node slows down players, ranging from 1 to 7,
+	// the higher, the slower.
+	f.move_resistance = getintfield_default(L, index,
+			"move_resistance", f.move_resistance);
```

Comment:
Value needs clamping, especially to zero before assigning to a `u8` variable.

Reply:
Clamping? I'd much prefer an error be thrown if a modder tries to set this to an invalid value.